src/_guides/libraries/create-library-packages.md

```diff
@@ -1,13 +1,11 @@
 ---
```

Comment:
Note the file name here ideally would change also. But if that will break links and things we would want a redirect. If we can't do that it might not be worth changing.